schemas/session-replay/mobile/image-wireframe-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/mobile/image-wireframe-schema.json",
+  "title": "ImageWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a ImageWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": ["type", "base64", "imageFileExtension"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "image",
+          "readOnly": true
+        },
+        "base64": {
+          "type": ["string", "null"],
+          "description": "base64 representation of the image. Can have a null value",
+          "readOnly": false
+        },
+        "imageFileExtension": {
+          "type": "string",
+          "description": "Extension of the image file",
+          "readOnly": false
+        }
+      }
```

Comment:
suggestion: may we add a unique identifier for the image? maybe optional for now.

Reply:
@acorretti it already has it as it inherits from `_common-wireframe-schema`

Reply:
I would say the wireframe would be the unique identifier for now. If we find a way (though the POC implementation) to reuse a base64, I will add this identifier in a follow up PR